labs/b9.md

```diff
@@ -0,0 +1,178 @@
+﻿---
```

Comment:
I'm surprised GitHub doesn't show the BOM at the start of the file, running `sed -i '1s/^\xEF\xBB\xBF//' labs/b9.md` fixed it for me though. (taken from https://unix.stackexchange.com/a/381263)